index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./index.css" rel="stylesheet" type="text/css">
+    <title>Awesome books</title>
+</head>
+<body>
+    <header>
+        <section class="heading">
+            <h1>Awesome Books</h1>
+        </section>
+        <nav>
+            <ul class="nav-bar">
+                <li class="item1"><a href="#">List</a></li>
+                <li class="item2"><a href="#">Add New</a></li>
+                <li class="item3"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <div id="current-time-zone">
+            <div id="date-time"></div>
+          </div>          
+    </header>
+    <main class="main-container">
+        <section class="content" id="listContent">
+            <h2>List Content</h2>
+            <div class="book-content">
+                <ul id="book-list" class="book-list"></ul>
+            </div>
+        </section>
```

Comment:
- [ ] Please adhere to the project requirements, **(It's true that you refactor your code from old awesome book project, but it is also true that your this milestone does contain all the requirements of the previous project )** according to which your design should match wireframe. As per the wireframe, when a person click on `List` navigation button, then a person should see the list of the books. When a person click on the `Add New` button then form should appear (only), with a help of which a person can add the new book. Kindly make sure that your List of books and **Add New** book form should not appear at the same page.

### Requirement (It is given in the **Awesome books: a complete website with navigation** milestone)
![image](https://github.com/DevKaranJ/AwesomeBooks-ES6/assets/66476329/6710b46c-e774-4ea5-ae0f-92d86cf7509a)


### Design Comparison
<table>
<tr>
<th>Awesome Book Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![awesome_books_full_website](https://github.com/DevKaranJ/AwesomeBooks-ES6/assets/66476329/9404035c-ccdf-4aad-8337-19e0d47cdf0c)

</td>
<td>

![Screenshot from 2023-08-22 14-19-56](https://github.com/DevKaranJ/AwesomeBooks-ES6/assets/66476329/067722b7-12f8-4e3a-971b-e1200d1af399)

</td>
</tr>
</table>